app/src/components/Preferences/ReviewPreferences.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Header from '../shared/Header';
+import PreferencesCard from './PreferencesCard';
+import { Button } from '../../store/index';
+
+export default function ReviewPreferences() {
+  return (
+    <>
+      <Header
+        title='Review Preferences'
+        subtitle='Review the information is correct before creating'
+      />
+      <PreferencesCard />
+      <div
+        style={{
+          position: 'absolute',
+          bottom: 0,
+          right: 0,
+        }}>
+        <Button size='large'>cancel</Button>
```

Comment:
These buttons feel a little cramped down in the corner based on the mockups in the Jira ticket. Probably just needs a little margin to give them some breathing room.